phpBB/styles/prosilver/theme/common.css

```diff
@@ -1266,7 +1300,8 @@ ul.linklist:after,
 	content: "";
 }
 
-.dropdown-extended .footer {
+.dropdown-extended .footer,
+.smiley-list .footer {
```

Comment:
just add the dropdown-extended and remove the smiley-list. the only difference is the padding which you can fix with one line rather than re-add 4 lines